packages/core-data/src/actions.js

```diff
@@ -81,3 +81,17 @@ export function receiveThemeSupportsFromIndex( index ) {
 		themeSupports: index.theme_supports,
 	};
 }
+
+/**
+ * Returns an action object used in signalling that Upload permissions have been received.
+ *
+ * @param  {boolean} uploadFiles Upload permissions.
```

Comment:
Minor: Don't need alignment of the `@param` with `@return`.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/

Reply:
Minor: The name of this variable could better indicate it holds a boolean value, e.g. `canUploadFiles`, or even `hasUploadPermissions` since that's what you're using for the action parameter anyways. No need to be inconsistent.